doc/news/changes/major/20190603PeterMunch

```diff
@@ -0,0 +1,7 @@
+New: The interface Utilities::MPI::ConsensusAlgorithm has been added. It is 
+implemented by Utilities::MPI::ConsensusAlgorithm_NBX and 
```

Comment:
I realize now that the combination of Pascal/camel case and underscore - I feel that I would prefer `ConsensusAlgorithmNBX`. Apart from the finite element classes like `FE_DGQ` where it makes sense because FE is already an abbreviation that we want to separate and some associated polynomials, the combination of underscores with capital letters is not very common in deal.II. As we have not released that class yet, we could still change it - any opinions on this?

Reply:
I am fine with that. It is mainly an internal change. I am normally using them directly, but use `ConsensusAlgorithmSelector` (which does already have the right name format).

Reply:
I will do the renaming. But lets merge #9600 first.

Reply:
Please rebase from master and update the name now that #9600 is done.